addons/website_sale_slides/data/website_sale_slides_demo.xml

```diff
@@ -17,4 +17,42 @@
         <field name="product_id" ref="product_course"/>
     </record>
 
+    <!-- CHANNEL 1: Taking care of Trees -->
+    <!-- ================================================== -->
+    <record id="water_can" model="product.product">
```

Comment:
Small remark: I generally vote for neutral IDs for demo. That way if we change the content of the demo we don't have to change its xml ID . For example here I would have something like product_course_channel_1_option_0 so that we know this is the first option of the product_course linked to channel_1 .